validate/automated/nodectlCommands.md

```diff
@@ -246,9 +246,9 @@ After a `restart_only` is executed, the profile should end in an `ReadyToJoin` s
 ---
 The **`leave`** command takes a single [parameter](#what-is-an-option-and-parameter).
 
-**Leave** the hypergraph or metagraphs related to a configured profile name.  This command will not work without the `<profile_name>` [parameter](#what-is-an-option-and-parameter) supplied.
+**Leave** the Hypergraph or Metagraphs related to a configured profile name.  This command will not work without the `<profile_name>` [parameter](#what-is-an-option-and-parameter) supplied.
```

Comment:
I see this throughout but I'll just comment here. "Hypergraph" is capitalized because it's a singular proper noun. "metagraph" should not be capitalized because it's a plural non-proper noun.